pkgs/top-level/all-packages.nix

```diff
@@ -13096,16 +13086,14 @@ with pkgs;
     })
     else if stdenv.isLinux
     then utillinuxMinimal
-    else null;
+    else libossp_uuid;
```

Comment:
I'm hoping these are completely compatible? Ideally I'd like libossp_uuid to be used everywhere but not sure if that will break something on Linux.

Reply:
I'll probably just remove this and put it in another PR.